portal_calendar.ino

```diff
@@ -117,7 +119,9 @@ void errorNoWifi()
         "or you entered the wrong password.",
         "",
         "WiFi Name:",
-        "\"" WIFI_NAME "\""
+        "\"" WIFI_NAME "\"",
+        "Hostname:",
+        "\"" HOSTNAME "\"",
```

Comment:
> What's the reason for showing hostname here?

No real reason. It seemed marginally useful. Happy to remove it if you think it's just clutter!

Reply:
I'd say go ahead and remove it. I don't think displaying the hostname there makes much sense for the average user.

Reply:
No problem, removed!